desafio-02/zezola/java/main.java

```diff
@@ -0,0 +1,21 @@
+public class NumerosPrimos {
+    public static void main(String[] args) {
+
+        for (int i = 2; i < 10000; i++) {
+            if (primeChecker(i)) {
+                System.out.println(i);
+            }
+        }
+
+    }
+
+    public static boolean primeChecker(int num) {
+        for (int i = 2; i < num; i++){
```

Comment:
Este loop acaba fazendo algumas comparações desnecessárias que podem impactar na performance do programa.

Por exemplo, se o número não é multiplo de 2, não será multiplo de 4, nem de 8, 16, 32, 64....

Do mesmo modo que se o número não for múltiplo de 3, não será de 9, 27...

Sugiro que dê uma lida [nestas duas primeiras linhas deste artigo da wikpedia](https://pt.wikipedia.org/wiki/N%C3%BAmero_primo#Os_%C3%A1tomos_da_aritm%C3%A9tica) que dá uma sugestão sobre como aplicar uma otimização para diminuir o range de números que este loop irá percorrer.